examples/webgl_morphtargets_face.html

```diff
@@ -95,7 +95,7 @@
 
 							gui.add( influences, value, 0, 1, 0.01 )
 								.name( key.replace( 'blendShape1.', '' ) )
-								.listen( influences );
+								.listen();
```

Comment:
See [docs for this method](https://lil-gui.georgealways.com/#Controller#listen).